Classes/Private/Manager/Database/OCTRealmManager.m

```diff
@@ -260,6 +292,31 @@ - (OCTMessageAbstract *)addMessageWithText:(NSString *)text
     return messageAbstract;
 }
 
+- (void)addMessageCall:(OCTMessageCallEvent)event
+                  call:(OCTCall *)call
+          callDuration:(NSTimeInterval)duration
+{
+    NSParameterAssert(event);
```

Comment:
I don't think that it is good idea to add assert for non-object parameter. If someone would pass `123` this won't fail, so there is not so much benefit from it.
Also it is more convenient to start NSENUM with 0, not 1.